dap-python.el

```diff
@@ -216,7 +218,12 @@ overriden in individual `dap-python' launch configurations."
          (plist-put conf :debugServer debug-port)
          (plist-put conf :port debug-port)
          (plist-put conf :hostName host)
-         (plist-put conf :host host)))
+         (plist-put conf :host host)
+         (when (and (not (null justmycode))
```

Comment:
```suggestion
         (when (and justmycode
```
Chaining `not` and `null` if only the result's truthynesss matters is redundant and non-idiomatic.

Reply:
FYI, `not` and `null` are aliases.